pyproject.toml

```diff
@@ -22,15 +22,6 @@ opentelemetry-instrumentation-django = "*"
 opentelemetry-exporter-richconsole = "*"
 opentelemetry-exporter-otlp-proto-http = "*"
 
-[tool.poetry.group.dev.dependencies]
-black = "*"
-isort = "*"
-pre-commit = "^3.0.0"
-pytest = "^8.0.0"
-twine = "^5.0.0"
-ruff = "^0.4.0"
-pytest-mock = "^3.14.0"
-
```

Comment:
Why did this block get deleted? We can legitimately get rid of black and isort, but the rest should be retained.

Reply:
I thought we were only using these dependencies in the CI, therefore deleted them, will re-add them in the next commit.